go/cmd/dolt/commands/sqlserver/server.go

```diff
@@ -271,8 +271,11 @@ func ConfigureServices(
 	var sqlEngine *engine.SqlEngine
 	InitSqlEngine := &svcs.AnonService{
 		InitF: func(ctx context.Context) (err error) {
-			if _, err := mrEnv.Config().GetString(env.SqlServerGlobalsPrefix + "." + dsess.DoltStatsBootstrapEnabled); err != nil {
-				// unless otherwise specified by config, enable server bootstrapping
+			if _, err := mrEnv.Config().GetString(env.SqlServerGlobalsPrefix + "." + dsess.DoltStatsAutoRefreshEnabled); err != nil {
```

Comment:
This logic is pretty weird. Variable is named "enabled" but we're treating it in the logic as "not disabled". Which is it? What is the default? Can it be set to "false"?

Reply:
ok, so just invert the naming now that we've inverted default behavior?

Reply:
On a second look, i think the confusing part is that we previously decided `sql-server` is the only context where we enable default stats behavior. So it is default off for everything but `sql-server`, and default on for `sql-server` unless the config explicitly specifies a behavior. I'm going to just document better and still ship the current behavior/naming.